packages/generator/templates/app/.env.test.local

```diff
@@ -1,7 +1,3 @@
 # THIS FILE SHOULD NOT BE CHECKED INTO YOUR VERSION CONTROL SYSTEM
 
-# SQLite is ready to go out of the box, but you can switch to Postgres
-# by first changing the provider from "sqlite" to "postgres" in the Prisma
-# schema file and by second swapping the DATABASE_URL below.
-DATABASE_URL="file:./db_test.sqlite"
-# DATABASE_URL=postgresql://__username__@localhost:5432/__name___test
+DATABASE_URL="__databaseUrl__"
```

Comment:
We definitely need to fix this to have the test db name with `_test` suffix.

Reply:
I was also thinking the same but left it like this per your comment on Slack. Going to change it 🙂

Reply:
Ah oops, I totally misunderstood your question there 😅